app/assets/javascripts/oxalis/model/helpers/reduce_reducers.js

```diff
@@ -9,7 +9,8 @@ import deepFreezeLib from "deep-freeze";
 // Do not use the deep-freeze library in production
 // process.env.NODE_ENV is being substituted by webpack
 let deepFreeze = deepFreezeLib;
-if (process.env.NODE_ENV === "production") deepFreeze = _.identity;
+// if (process.env.NODE_ENV === "production")
```

Comment:
We probably don't want to merge this, right? Or did the freezing cause heavy problems?